style.css

```diff
@@ -0,0 +1,165 @@
+body {
+	width: 100%;
+	height: 100%;
+	margin: 0 auto;
+	background: url(images/Background.png);
+	background-size: cover;
+}
+
+.wrap {
+	font-family: Roboto, serif;
+	color: #fff;
+	padding: 20px;
+	max-width: 1280px;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 50%;
+  margin: 0 auto;
+  transform: translate(0, -50%);
+}
+
+.title {
+	font-size: 57px;
+}
+
+.subtitle {
+	font-size: 19px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/evagame/pullRequest?prid=5788637)